scargo/jinja/base_gen.py

```diff
@@ -23,8 +24,8 @@ def create_file_from_template(
         self,
         template: str,
         output_path: Path,
+        template_params: Dict[str, Any],
```

Comment:
Mostly because I didn't like template parameters mixed with other arguments. It's cleaner to have them separated. Consider e.g. a typo in `overwrite`, it would silently fall through to `**kwargs`. Unfortunately it does not seem to be a standard way to add typing to Jinja2 templates, so any typos and wrong types will not be detected.

Reply:
if you don't like it I can try to remove it, but possibly there were some mypy errors otherwise, not sure now

Reply:
I don't like it, but if you think it's better we can keep it.